### PR TITLE
Add K8s 1.22 specific version information and build statuses to README

### PR DESCRIPTION
**Pull request by abhay-krishna**
Adding 1.22 specific build metadata to READMEs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: UPSTREAM_PROJECTS.yaml [hunk 1]
```diff
@@ -76,6 +76,7 @@ projects:
     repos:
       - name: cloud-provider-vsphere
         versions:
+          - tag: v1.22.5
```
Comment (abhay-krishna) [inline UPSTREAM_PROJECTS.yaml:79]: not sure why it got added at top

**Review by jaxesn — APPROVED** [review]
/lgtm

**Review by vivek-koppuru — APPROVED** [review]
/lgtm

**Pull request merged**